stock_picking_operation_quick_change/views/stock_picking_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
```

Comment:
2017-2018 (everywhere)

Reply:
why? Sergio didn't do anything this 2018 in this module or previous versions 🤔